src/render/locals.rs

```diff
@@ -332,7 +342,10 @@ pub fn print_ptr(
     ptr: Pointer<Tag>,
     size: Option<u64>,
 ) -> Result<(Option<u64>, String, u64), ()> {
-    match (ecx.memory.get_raw(ptr.alloc_id), ecx.memory.get_fn(ptr.into())) {
+    match (
+        ecx.memory.get_raw(ptr.alloc_id),
+        ecx.memory.get_fn(ptr.into()),
+    ) {
```

Comment:
Just like this

Reply:
we can probably tweak rustfmt.toml by increasing the line length or some other flags

Reply:
The old line was less than 100 characters. I can't find any option for this specific case.